packages/components/src/theme/navbar.scss

```diff
@@ -633,6 +633,7 @@
 
   .navbar__input-group {
       width: 251px;
+      display: table;
```

Comment:
this seems unrelated to the other changes in this PR, just curious what this was added/changed for.

Reply:
When showing the search input for the FM stand-alone app (which doesn't have the dropdown for the other find options), without this the box gets shifted down so it's not aligned in the navigation